### ISSUE TITLE
Hide/mark accounts that you already follow on Mastodon

### ISSUE THREAD

**Issue by pruvisto** [pruvisto/debirdify#8]
When querying the people you follow on Twitter, that follow you on Twitter, that you blocked on Twitter etc. it would be good to have an option to mark or even hide those that you already follow on Mastodon and exclude them from the generated CSV.

This necessitates interaction with the Mastodon API.

**Comment by cooljeanius** [comment]
Yeah I agree; getting my way through all of the accounts in my Debirdify output is going to be quite difficult without this

**Comment by eloquence** [comment]
This would be a wonderful feature whose value would continuously increase as folks use Debirdify repeatedly over the coming weeks to check for new mutuals :-)

**Comment by cooljeanius** [comment]
Kinda depends on #18 I guess
